modules/installation-gcp-service-account.adoc

```diff
@@ -6,24 +6,29 @@
 [id="installation-gcp-service-account_{context}"]
 = Creating a service account in GCP
 
-{product-title} requires a Google Cloud Platform (GCP) service account.
+{product-title} requires a Google Cloud Platform (GCP) service account that provides authentication and authorization to access data in the Google APIs. If you do not have an existing IAM service account that contains the required roles in your project, you must create one.
 
 .Prerequisites
 
 * You created a project to host your cluster.
 
 .Procedure
 
-. Create a new service account in the project that you use to host your
+. Create a service account in the project that you use to host your
 {product-title} cluster. See
 link:https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating_a_service_account[Creating a service account]
 in the GCP documentation.
 
 . Grant the service account the appropriate permissions. You can either
 grant the individual permissions that follow or assign the `Owner` role to it.
```

Comment:
> the individual permissions that follow

Seems to be missing the list of needed permissions, although I am an adocs newb and perhaps the list is composed from elsewhere.